documentapi/src/main/java/com/yahoo/documentapi/SyncSession.java

```diff
@@ -85,9 +84,8 @@ default Document get(DocumentId id, TemporalAmount timeout) {
      * @param timeout Timeout. If timeout is null, an unspecified default will be used.
      * @return The known document having this id, or null if there is no
      *         document having this id.
-     * @throws UnsupportedOperationException Thrown if this access does not
-     *                                       support retrieving.
-     * * @throws RuntimeException If operation times out.
+     * @throws UnsupportedOperationException Thrown if this access does not support retrieving.
+     * @throws DocumentAccessException with error code {@link com.yahoo.messagebus.ErrorCode#TIMEOUT} on timeout.
```

Comment:
Technically these methods will throw a `DocumentAccessException` upon _any_ error, not just timeout. Risk of users assuming™ that `DocumentAccessException` is only thrown for timeout errors. Perhaps either generalize the description or mention timeout error code as an addendum.